composables/use-reader.ts

```diff
@@ -1,7 +1,26 @@
-export default function () {
+export default function (
+  {
+    nftClassIdInput,
+  }: {
+    nftClassIdInput?: Ref<string> | string
+
+  } = {},
+) {
   const config = useRuntimeConfig()
 
-  const nftClassId = computed(() => getRouteQuery('nft_class_id'))
+  const nftClassId = computed(() => {
+    const queryValue = getRouteQuery('nft_class_id')
+    if (queryValue !== undefined && queryValue !== '') {
+      return queryValue
+    }
+    else if (nftClassIdInput) {
+      return toValue(nftClassIdInput)
+    }
+    else {
+      return ''
+    }
```

Comment:
'0' is trueful, just use `||` ?
i.e. `return queryValue || toValue(nftClassIdInput) || ''`

Reply:
[♻️ Simplify nftClassId and nftId computed properties](https://github.com/likecoin/3ook-com/pull/47/commits/15d0aa6f49f4ac18c081ab7a78fc117f2966fb5f)